### PR TITLE
Improve unconfigured state.

### PR DESCRIPTION
**Pull request by crspeller**
image

### CODE REVIEW

File: webapp/src/components/rhs/common.tsx [hunk 1]
```diff
@@ -29,3 +29,23 @@ export const Button = styled.button`
 		margin-right: 2px;
 	}
 `;
+
+export const RHSTitle = styled.div`
+    font-family: Metropolis;
+    font-weight: 600;
+    font-size: 22px;
```
Comment (matthewbirtch) [inline webapp/src/components/rhs/common.tsx:36]:
```suggestion
    font-size: 22px;
    line-height: 28px;
```

File: webapp/src/components/rhs/common.tsx [hunk 2]
```diff
@@ -29,3 +29,23 @@ export const Button = styled.button`
...
+`;
+
+export const RHSText = styled.div`
+    font-weight: 400;
```
Comment (matthewbirtch) [inline webapp/src/components/rhs/common.tsx]:
```suggestion
    font-weight: 500;
```

File: webapp/src/components/rhs/common.tsx [hunk 3]
```diff
@@ -29,3 +29,23 @@ export const Button = styled.button`
...
+    font-size: 14px;
```
Comment (matthewbirtch) [inline webapp/src/components/rhs/common.tsx:42]:
```suggestion
    font-size: 14px;
    line-height: 20px
```

File: webapp/src/components/rhs/rhs.tsx [hunk 4]
```diff
@@ -76,7 +89,22 @@ export default function RHS() {
...
+                <RHSPaddingContainer>
+                    <RHSImage/>
+                    <RHSTitle>{'Copilot is not yet configured for this workspace'}</RHSTitle>
+                    <RHSText>{'A system admin needs to complete the configuration before it can be used.'}</RHSText>
+                    <RHSDivider/>
+                    <RHSSubtitle>{'What is Copilot?'}</RHSSubtitle>
+                    <RHSText>{'Copilot is a plugin that enables you to leverage the power of AI to:'}</RHSText>
+                    <RHSText>
+                        <ul>
+                            <li>{'Get caught up quickly with instant summarization for channels and threads.'}</li>
```
Comment (matthewbirtch) [inline webapp/src/components/rhs/rhs.tsx]: @crspeller can we add `margin-bottom: 8px;` to each `<li>` item so they are spaced out a bit more?

**Review by matthewbirtch — APPROVED** [review]
Approved with the minor changes provided. If you wouldn't mind sharing an updated screenshot when completed.

**Comment by crspeller** [comment]
@matthewbirtch updated the picture in the description with the changes.

**Comment by matthewbirtch** [comment]
Thanks @crspeller. Good to go!

**Review by matthewbirtch — APPROVED** [review]

**Pull request merged**